src/components/modals/Dialog.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useLayoutEffect } from 'react';
+import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
+import { CSSTransition } from 'react-transition-group';
+import FocusTrap from 'focus-trap-react';
+import noScroll from 'no-scroll';
+import DenyIcon from '../../../lib/icons/react/DenyIcon';
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import Section from '../layout/Section';
+
+const Dialog = (props) => {
+  const handleClick = () => {
+    if (props.canDismiss) {
+      props.onDismiss();
+    }
+  };
+
+  const handleKeyDown = (e) => {
+    if (props.canDismiss && e.key === 'Escape') {
```

Comment:
does babel lend any help with this in the transpiled component? `e.key` may need to be polyfilled

Reply:
so from what I researched, e.key is supported even up until IE9.... it's just that IE has non-standard key values... so you'll have people doing stuff like `e.key === "ArrowDown" || e.key === "Down`. All the polyfill things I saw where slightly underwhelming, and from reading the docs, `Escape` seemed supported on IE as well, but haven't had a chance to check. Let me get Browserstack up and running to see if this works on IE11.